manifests/config.pp

```diff
@@ -1,28 +1,28 @@
-# == Class: xwindows::gdm
-#
-# This class installs gdm and sets up the DESKTOP environment variable and
-# /etc/gconf.
+# == Class gdm::config
 #
 # == Parameters
 #
 # [*include_sec*]
-#   Whether or not to include the bult in 'sec' class.  Defaults to true.
+# Boolean
+# Whether or not to include ::gdm::sec
+# Defaults to true
 #
 # == Authors
 #
 # * Trevor Vaughan <tvaughan@onyxpoint.com>
+# * Nick Markowski <nmarkowski@keywcorp.com>
 #
-class xwindows::gdm(
+class gdm::config(
```

Comment:
This shouldn't be here. Just roll it into the main class with all of the other selectors.